packages/db/tests/search/test-GranuleSearch.js

```diff
@@ -859,3 +931,91 @@ test('GranuleSearch estimates the rowcount of the table by default', async (t) =
   t.true(response.meta.count > 0);
   t.is(response.results?.length, 50);
 });
+
+test('GranuleSearch with includeFullRecord true retrieves associated file objects for granules', async (t) => {
```

Comment:
Nit: Is this test a duplication of the test on L952 (in that L935 is a less specific case that is covered by 952 and the more general case doesn't test anything more/new/different)?

Reply:
the intention of this distinction is specificity: if files fail to be retrieved, that will show up as one error and if files fail to be translated to api format that will show up as a distinct error